### PR TITLE
Document release procedure. Fixes #216

### LINKED ISSUE #216: Document release procedure
Describing regular procedure and hotfixing.

### PR DESCRIPTION
**Pull request by rdebusscher**

### CODE REVIEW

File: releasing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Procedures around PRs and releases
```
Comment (cealsair) [inline releasing.md]: Suggestion mod as follows:
"Procedures around Pull Requests (PRs) and releases

File: releasing.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Releases on the [test site](https://test-start.microprofile.io/) are performed automatically each time a PR is merged into the master branch.
+
+- Changes are pulled regularly into [repo](https://github.com/Karm/microprofile-starter/).
+- Circle CI performs build and push to the server.
+- Message is pushed to the [gitter chat](https://gitter.im/eclipse/microprofile-starter) when updates are live.
+
+## Releases on production
+
+Releasing on production requires a manual step and can only be performed by those who have the SSH key for the production machine.
```
Comment (cealsair) [inline releasing.md]: Add sentence:
" The following commands will build and release to production the currently checked-out tag of the MicroProfile Starter:"

**Comment by Karm** [comment]
@cealsair @Emily-Jiang  can you ack?

File: releasing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+The following commands will build and release to production the currently checked-out tag of the MicroProfile Starter:
+
+- mvn package -Pthorntail
+- unzip target/mp-starter-hollow-thorntail.jar -d target/mp-starter-hollow-thorntail
+- docker build -f Container/Dockerfile -t microprofile/start.microprofile.io:_1.0_ .
```
Comment (Emily-Jiang) [inline releasing.md]: There should be an explaination on _1.0_. This version will change and the version should be sequential number after the current release.

Comment (rdebusscher) [reply releasing.md]: @Emily-Jiang Indicated we will follow semantic versioning rules.

**Pull request merged**